services/logs-forwarder/.lagoon.multi.yml

```diff
@@ -259,19 +261,17 @@ objects:
           ssl_version TLSv1_2
           request_timeout 60s
           slow_flush_log_threshold 90s
-          <buffer tag,time>
+          <buffer>
             @type file
             path /fluentd/buffer/elasticsarch
-            timekey 3600
-            timekey_wait 0s
-            flush_mode interval
-            flush_interval 10s
-            chunk_limit_size 16MB
+            # buffer params
+            chunk_limit_size 32MB
             total_limit_size 15GB
-            flush_thread_count 8
-            overflow_action block
-            retry_type periodic
-            retry_wait 10s
+            # flush params
+            flush_mode interval
+            flush_interval 60s
+            flush_thread_count 16
+            overflow_action drop_oldest_chunk
```

Comment:
my understanding of having `overflow_action block` is described here: https://docs.fluentd.org/configuration/buffer-section

> block processing of input plugin to emit events into that buffer

basically it means that fluentd tells the inputs (which are fluentds again in logs-collector and external logs-forwarder) to not send data anymore, so it will will the buffers of these fluentds as well.
I think that's a bit safer than just dropping the oldest chunks?

Reply:
Well I guess this a philosophical debate :smile: 

I think that the chunks should be dropped at the point that the bottleneck is occurring because that makes it easy to identify the point in the log pipeline that the issue is located: it's the only fluentd with a full buffer and a log full of dropped chunk warnings.

If you `block` instead, the entire pipeline backs up which tends to obfuscate the location of the issue because buffers are full everywhere and you may see issues at points in the pipeline several hops removed from the actual bottleneck. You will still end up dropping logs eventually - it just take slightly longer and will be further back in the pipeline.